cl/scrapers/tests.py

```diff
@@ -438,3 +443,41 @@ def test_process_audio_file(self):
             msg="We should end up with the proper duration of about %s. "
             "Instead we got %s." % (expected_duration, measured_duration),
         )
+
+
+class BinaryTransformerExtractionTest(TestCase):
+    def setUp(self):
+        self.path = os.path.join(settings.MEDIA_ROOT, "test", "search")
+
+    def test_heartbeat(self):
+        """Can we start container and check sanity test?"""
+        base_url = "http://cl-binary-transformers-and-extractors:80"
+        response = requests.get(url=base_url, timeout=60).json()
+        self.assertTrue(response["success"], msg="Failed heartbeat test.")
+        print(response)
+
+    def test_content_extraction(self):
+        """Do all of the supported mimetypes get extracted to text
+        successfully, including OCR?"""
+        test_strings = [
+            "supreme",
+            "indiana",
+            "reagan",
+            "intelligence",
+            "october",
+            "tax court",
+            "fidelity",
+            "indiana",
+            "vermont",
+        ]
+        opinions = sorted(iglob(os.path.join(self.path, "*")))
+        for filepath, test_string in zip(opinions, test_strings):
```

Comment:
Rework your data structure here to pair the words with the correct files. The whole thing works b/c of the order of the list and the names of the files, but that's way too fragile. Gotta be a better way.

Reply:
I would prefer to just add the answers to a test assets directory and compare the results.  Perhaps you could take a look at the tests in the binary transformers extractors image repo and see if you like those tests?

Reply:
Mind pointing me at a specific one in the transformers so I'm sure what you're suggesting? (Sorry if my earlier comment was a bit terse.)

Reply:
@mlissner https://github.com/freelawproject/courtlistener-binary-transformers-and-extractors/blob/b53013c42be792c28ce30b81e02158f7ee7c20cc/tests/tests.py#L113. this entire class

Reply:
Yeah, I think the bte approach is a step in the right direction (it doesn't depend on the alphabetical ordering of a directory to work), but I still find it pretty tricky to work with. You have to go from `DocumentConversionTests` to its super class, `DockerTestBase`, to a json file it creates, through the code that converts it, then to the key in that file, and finally you know what the test is being compared to. Wouldn't it be easier if those tests just had something like juriscraper does:

    some_file.pdf
    some_file_result.txt

And then you just grab the contents of `some_file_result.{txt,json,etc}` when doing your assertion? 

---

For these tests though, it occurs to me (belatedly, sorry) that the only thing we need to test is that the boundaries work. That's the beauty of the microservices. In BTE, we test that conversions happen properly; we don't need to *also* test that here every time we make a one-line tweak to the code. 

All we need to make sure is that they're properly compatible at the boundaries. Maybe just send one really basic request to each endpoint, and make sure it returns a 200 or something? (I'm assuming the endpoints will fail if they get parameters they don't like?)